[rocm-opencl-runtime] rocclr include paths not found

### Information

I cannot build the current version of rocm-opencl-runtime (3.9.0-1). The error message looks as follows:

```
Scanning dependencies of target OpenCL
Scanning dependencies of target Common
[  4%] Building C object khronos/icd/CMakeFiles/OpenCL.dir/loader/icd.c.o
[  4%] Building C object khronos/icd/CMakeFiles/OpenCL.dir/loader/linux/icd_linux.c.o
[  8%] Building C object khronos/icd/CMakeFiles/OpenCL.dir/loader/icd_dispatch.c.o
[ 12%] Building C object khronos/icd/CMakeFiles/OpenCL.dir/loader/linux/icd_linux_envvars.c.o
[ 12%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/BaseTestImp.cpp.o
[ 16%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/OCLGLCommon.cpp.o
[ 16%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/OCLGLCommonLinux.cpp.o
[ 20%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/OCLTestImp.cpp.o
[ 25%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/OCLTestUtils.cpp.o
[ 25%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/OCLTestListImp.cpp.o
[ 25%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/OCLThread.cpp.o
[ 25%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/Timer.cpp.o
[ 29%] Building CXX object tests/ocltst/module/common/CMakeFiles/Common.dir/OCLWrapper.cpp.o
Scanning dependencies of target amdocl64
[ 33%] Building CXX object amdocl/CMakeFiles/amdocl64.dir/cl_memobj.cpp.o
[ 33%] Building CXX object amdocl/CMakeFiles/amdocl64.dir/cl_program.cpp.o
[ 37%] Building CXX object amdocl/CMakeFiles/amdocl64.dir/cl_sdi_amd.cpp.o
[ 37%] Building CXX object amdocl/CMakeFiles/amdocl64.dir/cl_device.cpp.o
[ 41%] Building CXX object amdocl/CMakeFiles/amdocl64.dir/cl_debugger_amd.cpp.o
[ 45%] Building CXX object amdocl/CMakeFiles/amdocl64.dir/cl_svm.cpp.o
In Datei, eingebunden von /opt/rocm/rocclr/include/elf/elf.hpp:31,
                 von /opt/rocm/rocclr/include/device/device.hpp:32,
                 von /opt/rocm/rocclr/include/platform/context.hpp:25,
                 von /opt/rocm/rocclr/include/platform/command.hpp:36,
                 von /opt/rocm/rocclr/include/vdi_common.hpp:26,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_common.hpp:32,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_sdi_amd.cpp:21:
/opt/rocm/rocclr/include/elf/elfio/elfio.hpp:42:10: schwerwiegender Fehler: elfio/elf_types.hpp: Datei oder Verzeichnis nicht gefunden
   42 | #include <elfio/elf_types.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~
Kompilierung beendet.
In Datei, eingebunden von /opt/rocm/rocclr/include/elf/elf.hpp:31,
                 von /opt/rocm/rocclr/include/device/device.hpp:32,
                 von /opt/rocm/rocclr/include/platform/context.hpp:25,
                 von /opt/rocm/rocclr/include/platform/command.hpp:36,
                 von /opt/rocm/rocclr/include/vdi_common.hpp:26,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_common.hpp:32,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_memobj.cpp:21:
/opt/rocm/rocclr/include/elf/elfio/elfio.hpp:42:10: schwerwiegender Fehler: elfio/elf_types.hpp: Datei oder Verzeichnis nicht gefunden
   42 | #include <elfio/elf_types.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~
Kompilierung beendet.
make[2]: *** [amdocl/CMakeFiles/amdocl64.dir/build.make:108: amdocl/CMakeFiles/amdocl64.dir/cl_sdi_amd.cpp.o] Fehler 1
make[2]: *** Es wird auf noch nicht beendete Prozesse gewartet....
make[2]: *** [amdocl/CMakeFiles/amdocl64.dir/build.make:82: amdocl/CMakeFiles/amdocl64.dir/cl_memobj.cpp.o] Fehler 1
In Datei, eingebunden von /opt/rocm/rocclr/include/elf/elf.hpp:31,
                 von /opt/rocm/rocclr/include/device/device.hpp:32,
                 von /opt/rocm/rocclr/include/platform/context.hpp:25,
                 von /opt/rocm/rocclr/include/platform/command.hpp:36,
                 von /opt/rocm/rocclr/include/vdi_common.hpp:26,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_common.hpp:32,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_program.cpp:21:
/opt/rocm/rocclr/include/elf/elfio/elfio.hpp:42:10: schwerwiegender Fehler: elfio/elf_types.hpp: Datei oder Verzeichnis nicht gefunden
   42 | #include <elfio/elf_types.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~
Kompilierung beendet.
make[2]: *** [amdocl/CMakeFiles/amdocl64.dir/build.make:95: amdocl/CMakeFiles/amdocl64.dir/cl_program.cpp.o] Fehler 1
In Datei, eingebunden von /opt/rocm/rocclr/include/elf/elf.hpp:31,
                 von /opt/rocm/rocclr/include/device/device.hpp:32,
                 von /opt/rocm/rocclr/include/platform/context.hpp:25,
                 von /opt/rocm/rocclr/include/platform/command.hpp:36,
                 von /opt/rocm/rocclr/include/vdi_common.hpp:26,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_common.hpp:32,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_device.cpp:21:
/opt/rocm/rocclr/include/elf/elfio/elfio.hpp:42:10: schwerwiegender Fehler: elfio/elf_types.hpp: Datei oder Verzeichnis nicht gefunden
   42 | #include <elfio/elf_types.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~
Kompilierung beendet.
make[2]: *** [amdocl/CMakeFiles/amdocl64.dir/build.make:121: amdocl/CMakeFiles/amdocl64.dir/cl_device.cpp.o] Fehler 1
In Datei, eingebunden von /opt/rocm/rocclr/include/elf/elf.hpp:31,
                 von /opt/rocm/rocclr/include/device/device.hpp:32,
                 von /opt/rocm/rocclr/include/platform/context.hpp:25,
                 von /opt/rocm/rocclr/include/platform/command.hpp:36,
                 von /opt/rocm/rocclr/include/vdi_common.hpp:26,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_common.hpp:32,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_debugger_amd.cpp:21:
/opt/rocm/rocclr/include/elf/elfio/elfio.hpp:42:10: schwerwiegender Fehler: elfio/elf_types.hpp: Datei oder Verzeichnis nicht gefunden
   42 | #include <elfio/elf_types.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~
Kompilierung beendet.
make[2]: *** [amdocl/CMakeFiles/amdocl64.dir/build.make:134: amdocl/CMakeFiles/amdocl64.dir/cl_debugger_amd.cpp.o] Fehler 1
In Datei, eingebunden von /opt/rocm/rocclr/include/elf/elf.hpp:31,
                 von /opt/rocm/rocclr/include/device/device.hpp:32,
                 von /opt/rocm/rocclr/include/platform/context.hpp:25,
                 von /opt/rocm/rocclr/include/platform/command.hpp:36,
                 von /opt/rocm/rocclr/include/vdi_common.hpp:26,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_common.hpp:32,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/amdocl/cl_svm.cpp:21:
/opt/rocm/rocclr/include/elf/elfio/elfio.hpp:42:10: schwerwiegender Fehler: elfio/elf_types.hpp: Datei oder Verzeichnis nicht gefunden
   42 | #include <elfio/elf_types.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~
Kompilierung beendet.
make[2]: *** [amdocl/CMakeFiles/amdocl64.dir/build.make:147: amdocl/CMakeFiles/amdocl64.dir/cl_svm.cpp.o] Fehler 1
make[1]: *** [CMakeFiles/Makefile2:557: amdocl/CMakeFiles/amdocl64.dir/all] Fehler 2
make[1]: *** Es wird auf noch nicht beendete Prozesse gewartet....
/var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/tests/ocltst/module/common/OCLWrapper.cpp: In Elementfunktion »_cl_command_queue* OCLWrapper::clCreateCommandQueue(cl_context, cl_device_id, cl_command_queue_properties, cl_int*)«:
/var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/tests/ocltst/module/common/OCLWrapper.cpp:184:14: Warnung: »_cl_command_queue* clCreateCommandQueue(cl_context, cl_device_id, cl_command_queue_properties, cl_int*)« ist veraltet [-Wdeprecated-declarations]
  184 |     return ::clCreateCommandQueue(context, device, properties, errcode_ret);
      |              ^~~~~~~~~~~~~~~~~~~~
In Datei, eingebunden von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/tests/ocltst/include/OCLWrapper.h:26,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/tests/ocltst/module/common/OCLWrapper.cpp:21:
/var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/khronos/headers/opencl2.2/CL/cl.h:1813:1: Anmerkung: hier deklariert
 1813 | clCreateCommandQueue(cl_context                     context,
      | ^~~~~~~~~~~~~~~~~~~~
/var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/tests/ocltst/module/common/OCLWrapper.cpp:184:75: Warnung: »_cl_command_queue* clCreateCommandQueue(cl_context, cl_device_id, cl_command_queue_properties, cl_int*)« ist veraltet [-Wdeprecated-declarations]
  184 |     return ::clCreateCommandQueue(context, device, properties, errcode_ret);
      |                                                                           ^
In Datei, eingebunden von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/tests/ocltst/include/OCLWrapper.h:26,
                 von /var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/tests/ocltst/module/common/OCLWrapper.cpp:21:
/var/tmp/pamac-build-sebastian/rocm-opencl-runtime/src/ROCm-OpenCL-Runtime-rocm-3.9.0/khronos/headers/opencl2.2/CL/cl.h:1813:1: Anmerkung: hier deklariert
 1813 | clCreateCommandQueue(cl_context                     context,
      | ^~~~~~~~~~~~~~~~~~~~
[ 45%] Linking C shared library libOpenCL.so
[ 45%] Built target OpenCL
[ 45%] Built target Common
make: *** [Makefile:149: all] Fehler 2
==> FEHLER: Ein Fehler geschah in build().
    Breche ab...
```

### System Information

- Operating System: Arch Linux
- KDE Plasma Version: 5.20.2
- KDE Frameworks Version: 5.75.0
- Qt Version: 5.15.1
- Kernel Version: 5.9.2-zen1-1-zen
- OS Type: 64-bit
- Processors: 16 × AMD Ryzen 7 1700X Eight-Core Processor
- Memory: 32 GB of RAM
- Graphics Processor: Radeon RX Vega 56

--- a/rocm-opencl-runtime/.SRCINFO
+++ b/rocm-opencl-runtime/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = rocm-opencl-runtime
 	pkgdesc = Radeon Open Compute - OpenCL runtime
 	pkgver = 3.9.0
-	pkgrel = 1
+	pkgrel = 2
 	url = https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime
 	arch = x86_64
 	license = MIT

--- a/rocm-opencl-runtime/PKGBUILD
+++ b/rocm-opencl-runtime/PKGBUILD
@@ -3,7 +3,7 @@
 
 pkgname=rocm-opencl-runtime
 pkgver=3.9.0
-pkgrel=1
+pkgrel=2
 pkgdesc='Radeon Open Compute - OpenCL runtime'
 arch=('x86_64')
 url='https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime'
@@ -17,8 +17,8 @@ sha256sums=('286ff64304905384ce524cd8794c28aee216befd6c9267d4187a12e5a21e2daf')
 _dirname="$(basename "$url")-$(basename "${source[0]}" .tar.gz)"
 
 build() {
-    CFLAGS="$CFLAGS -isystem /opt/rocm/rocclr/include/include -isystem /opt/rocm/rocclr/include/compiler/lib -isystem /opt/rocm/rocclr/include/compiler/lib/include" \
-    CXXFLAGS="$CXXFLAGS -isystem /opt/rocm/rocclr/include/include -isystem /opt/rocm/rocclr/include/compiler/lib -isystem /opt/rocm/rocclr/include/compiler/lib/include" \
+    CFLAGS="$CFLAGS -isystem /opt/rocm/rocclr/include/include -isystem /opt/rocm/rocclr/include/compiler/lib -isystem /opt/rocm/rocclr/include/compiler/lib/include -isystem /opt/rocm/rocclr/include/elf" \
+    CXXFLAGS="$CXXFLAGS -isystem /opt/rocm/rocclr/include/include -isystem /opt/rocm/rocclr/include/compiler/lib -isystem /opt/rocm/rocclr/include/compiler/lib/include -isystem /opt/rocm/rocclr/include/elf" \
     cmake -DCMAKE_INSTALL_PREFIX=/opt/rocm \
           -DROCclr_DIR=/opt/rocm/rocclr \
           -DLIBROCclr_STATIC_DIR=/opt/rocm/rocclr/lib \

